src/Directory.Packages.props

```diff
@@ -20,10 +21,10 @@
 		<PackageVersion Include="Microsoft.Maui.Controls.Compatibility" Version="8.0.3" />
 		<PackageVersion Include="Microsoft.NET.Test.Sdk" Version="17.6.2" />
 		<PackageVersion Include="Microsoft.Net.Compilers.Toolset" Version="4.2.0" />
-		<PackageVersion Include="Microsoft.SourceLink.GitHub" Version="1.1.1" />
+		<PackageVersion Include="Microsoft.SourceLink.GitHub" Version="8.0.0" />
 		<PackageVersion Include="Microsoft.UI.Xaml" Version="2.7.1" />
-		<PackageVersion Include="Microsoft.Windows.SDK.BuildTools" Version="10.0.22621.756" />
-	    <PackageVersion Include="Microsoft.WindowsAppSDK" Version="1.4.231219000" />
+		<PackageVersion Include="Microsoft.Windows.SDK.BuildTools" Version="10.0.22621.3233" />
+		<PackageVersion Include="Microsoft.WindowsAppSDK" Version="1.5.240311000" />
```

Comment:
Not sure if we want to bump this for now. We had a discussion with @nickrandolph around some issues that are "partially" on WinUI side and are fixed in 1.5, but I'm not sure if we'll end up wanting to update or not.